nms/app/packages/magmalte/app/components/feg/FEGGatewayDialog.js

```diff
@@ -146,8 +159,11 @@ export default function FEGGatewayDialog(props: Props) {
   const {match} = useRouter();
   const enqueueSnackbar = useEnqueueSnackbar();
 
-  const {editingGateway} = props;
-  const [tab, setTab] = useState(editingGateway ? 'gx' : 'general');
+  const ctx = useContext(FEGGatewayContext);
+  const {editingGateway, tabOption} = props;
+  const [tab, setTab] = useState(
+    editingGateway ? tabOption ?? 'gx' : 'general',
```

Comment:
nit: use the enum you created

Reply:
Oh, my bad. I forgot to edit that once I created the enum.